File: JAVA/Daily Flash Codes/Week2/Day6_Jan25/code/prog5.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int num1,num2;
		System.out.println("Enter range to print odd num: ");
		System.out.println("Enter num1: ");
		num1 = 	Integer.parseInt(in.readLine());
		
		System.out.println("Enter num2: ");
		num2 = 	Integer.parseInt(in.readLine());
			

		if(num1 > num2)
		       	System.out.print(num1); 
		else
			System.out.print(num2);	
		System.out.println(" is maximun among "+ num1 + " and "+ num2);
	 
	}


}

File: C/Daily Flash/Week2/Day6_Jan26/code/prog5.c
#include <stdio.h>

void main() {
	int num1,num2;
	printf("Enter num1: ");
	scanf("%d",&num1);
	
	printf("Enter num2: ");
	scanf("%d",&num2);

	num1 > num2 ? printf("%d ",num1): printf("%d ",num2);
	printf("is the maximum among %d and %d\n",num1,num2);


}
